internal/handlers/commoncommands/msg_buildinfo.go

```diff
@@ -45,6 +47,9 @@ func MsgBuildInfo(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
 
 			// our extensions
 			"ferretdbVersion", version.Get().Version,
+			"compatibility", must.NotFail(types.NewDocument(
```

Comment:
Let's rename it to `ferretdbFeatures` to avoid a potential name clash. I updated the issue